### DISCUSSION TITLE
Hide the connect button

### DISCUSSION THREAD

**Discussion by mveteanu** [microsoft/jacdac#1218] [General]
Please excuse my ignorance ... I didn't study long enough the code.
My question is: is there any way to hide the connect button that appears in a p5.js sketch?

I would like to control everything through code only.

Context: I'm considering this library for my coding project at https://codeguppy.com -- which is based on p5.js

**Comment by pelikhan** [comment]
I've exported the button instances in the latest build (1.8.6) that would allow you to hide them? https://github.com/microsoft/jacdac-ts/blob/main/src/p5/p5.jacdac.ts#L41

**Reply by pelikhan** [reply]
Another option would be to spin up your own flavor of p5.jacdac.ts to match codeguppy.com coding style.
